addons/popochiu/engine/objects/gui/templates/simple_click/components/simple_click_settings_popup/simple_click_settings_popup.gd

```diff
@@ -0,0 +1,54 @@
+@tool
+extends PopochiuPopup
+
+@onready var save: Button = %Save
+@onready var load: Button = %Load
+@onready var history: Button = %History
+@onready var quit: Button = %Quit
+@onready var text_speed: HSlider = %TextSpeed
+@onready var text_speed_label: Label = %TextSpeedLabel
+
+
+#region Godot ######################################################################################
+func _ready() -> void:
+	super()
+	
+	if Engine.is_editor_hint(): return
+	
+	# Set default values
+	text_speed.value = 0.1 - PopochiuUtils.e.text_speed
+	text_speed_label.text = "%.2fx" % PopochiuUtils.e.text_speed
```

Comment:
Unsure if these can be constants too.

Yes, today is constants day here :joy:

Reply:
Sure, why not. I'll do it.